hedera-node/src/test/java/com/hedera/services/txns/token/TokenWipeTransitionLogicTest.java

```diff
@@ -58,20 +64,28 @@
     private TokenStore tokenStore;
     private TransactionContext txnCtx;
     private PlatformTxnAccessor accessor;
-    private MerkleToken token;
+    private MerkleToken merkleToken;
+	private Token token;
 
     private TransactionBody tokenWipeTxn;
     private TokenWipeTransitionLogic subject;
+	private TypedTokenStore typedTokenStore;
+	private AccountStore accountStore;
+	private OptionValidator validator;
 
     @BeforeEach
     private void setup() {
```

Comment:
Lets cover the TransitionLogic in unit tests fully once we refactor the fungible minting as-well